lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -514,6 +514,9 @@ sub to_hash {
     }
     if ($args{deps}) {
         $j = {%$j, %{$job->deps_hash}};
+        my $dependencies = $job->dependencies;
+        $j->{has_parents} = $dependencies->{has_parents};
+        $j->{parents_ok} = $dependencies->{parents_ok};
```

Comment:
`$job->deps_hash` and `$job->dependencies` are very similar and hence invoking both functions means doing things twice. So it would be more efficient and cleaner to invoke only one of them. `$job->dependencies` also has `parents` and `children` and it might be possible to simply use `$j = $job->dependencies`.

Reply:
I'll see if I can get rid of `deps_hash`